docs/source/python/install.rst

```diff
@@ -63,6 +63,25 @@ need to install the `Visual C++ Redistributable for Visual Studio 2015
 .. warning::
    On Linux, you will need pip >= 19.0 to detect the prebuilt binary packages.
 
+You may encounter problems writing datetime data to a file if you install
+pyarrow with pip.  One process to fix this problem:
+
+   1. Install tzdata with ``pip install tzdata``
+
+   2. Set the environmental variable ``TZDIR`` = ``path/to/tzdata``
```

Comment:
I think it would be clearer to use `path\to\.venv` here directly. That way, the additional explanation on line 82 wouldn't be necessary. Also, since this only applies to Windows, we should use backslashes (`\`) instead of forward slashes (`/`).

```suggestion
   2. Set the environmental variable ``TZDIR`` = ``path\to\.venv\Lib\site-packages\tzdata\``
```